molgenis-searchall/src/main/java/org/molgenis/searchall/model/PackageResult.java

```diff
@@ -0,0 +1,21 @@
+package org.molgenis.searchall.model;
+
+import com.google.auto.value.AutoValue;
+import org.molgenis.gson.AutoGson;
+
+import javax.annotation.Nullable;
+
+@AutoValue
+@AutoGson(autoValueClass = AutoValue_PackageResult.class)
+public abstract class PackageResult
+{
+	abstract String getLabel();
```

Comment:
How can the search result jump to the package if it only knows the label. I think it should also contain the ID?